pyoverkiz/const.py

```diff
@@ -72,3 +76,9 @@
         configuration_url=None,
     ),
 }
+
+CLIENTS: dict[str, OverkizClient] = {
```

Comment:
Would be good to make OverkizClient part of the OverkizServer implementation? To have everything in one place, and even allow developers to pass their own OverkizServer object, with a custom Client.

Reply:
Done with e3f2461